File: javascript/series.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
   importPackage(java.io);
   importPackage(java.lang);
   var stdin = new BufferedReader(new InputStreamReader(System['in']));
   return function() {
     return String(stdin.readLine());   
   };                                    
 }());
print("Upto how many series do you want to print?");
count = readln();
var ref = "";
var term1 = 1;
var Termcounter = 1;
var term2 = (Termcounter + 1);
var final = 1;
while(final <= count)
{
  for(Loopcounter = 1; Loopcounter <= Termcounter; Loopcounter++)
  {
    term1 = Termcounter * term1;
    term2 = term2 * (Termcounter + 1) ;
  }
  final = term2;
  ref +=  " " + term1 ;
  term1 = 1;
  Termcounter += 1;
  term2 = (Termcounter + 1);
}
print("The series upto " + count + " are " + ref + ".");

File: c/series.c
/* Hello World program */

#include<stdio.h>
int main(void) 
{
 int Count, TermCounter = 2, Product, Counter;
 printf("Enter the number upto which you want to print the series:");
 scanf("%d", &Count);
 printf("The series upto %d is 1", Count);
 for(TermCounter = 2; Product <= Count; TermCounter++)
 {
   Product = TermCounter;
   for(Counter = 1; Counter < TermCounter; Counter++)
   {
     Product = Product * TermCounter;
   }
   if(Product < Count)
   {
     printf(", %d", Product);
   }
 }
 printf(".\n"); 
 return 0;
}


